content/stories/citrix.md

```diff
@@ -0,0 +1,118 @@
+
+When considering what to personalize, Citrix focuses on target audiences that align with company goals, produce enough website traffic to test on, and for which they can easily create personalized content. They use Demandbase to access data on their visitors (industry, demographic, location, etc.) and then filter than data into Optimizely to deliver and test each personalized experience. 
+
+The following example looks at a vertical-based personalization campaign that had a tremendous impact on Citrix’s business. 
+
+## HOMEPAGE PERSONALIZATION ON CITRIX.COM
+
+The Citrix team hypothesized that industry-specific content personalization on the Citrix.com homepage would increase engagment and decrease overall bounce rate, because visitors would see content relevant to their individual business needs. 
+
+Companies in the healthcare, education, and finance industries fit well into Citrix’s target audience, so they opted to run a test on these three audiences. 
+
+They created personalized banners on on Citrix.com, with copy, imagery, and a CTA that was specific to each vertical. Then, they built a test to measure the ROI of personalized content on these key audience segments. Half of each audience saw the personalized version, and half saw the original, “one-size-fits-all” version of Citrix’s homepage. 
+
+{% include "figure.html"
+  {
+    image: 'https://d1qmdf3vop2l07.cloudfront.net/optimizely-marketer-assets.cloudvent.net/raw/customer-stories/customer-stories-pages/citrix/citrix-financial.png',
+    caption: 'Citrix created this personalized homepage experience for visitors in the finance industry.',
+  }
+%}
+
+{% include "figure.html"
+  {
+    image: 'https://d1qmdf3vop2l07.cloudfront.net/optimizely-marketer-assets.cloudvent.net/raw/customer-stories/customer-stories-pages/citrix/citrix-medical.png',
+    caption: 'Citrix created this personalized homepage experience for visitors in the healthcare industry.',
+  }
+%}
+
+{% include "figure.html"
+  {
+    image: 'https://d1qmdf3vop2l07.cloudfront.net/optimizely-marketer-assets.cloudvent.net/raw/customer-stories/customer-stories-pages/citrix/citrix-education.png',
+    caption: 'Citrix created this personalized homepage experience for visitors in the education industry.',
+  }
+%}
+
+## RESULTS
+
+The results were staggering. The personalized experiences accounted for huge improvements in visitor engagement across all three target industries. 
+
+*Personalization of Citrix’s homepage resulted in*:*
+
+- An overall decrease in bounce rate by 7%
+- 30% increase in clicks on the homepage banners
+- 10% increase in pageviews per session, indicating that visitors were navigating to more pages after they saw a more personalized homepage experience
+- 4% increase in average session duration, indicating that not only were visitors seeing more pages, but they were also spending more time on the site and consuming more information 
+
+*average results across the three personalization campaigns
```

Comment:
This is actually just an asterisk to indicate that this sentence is a sidenote. Not supposed to be part of formatting.